packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -646,6 +649,14 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 				}
 			}
 		} finally {
+			if (controller.signal.aborted) {
+				this.logger.sendTelemetryEvent({
+					eventName: "DeltaManager_GetDeltasAborted",
+					fetchReason,
+					// Remove case to any when @types/node is upgraded to 16 (AB#4884)
+					reason: (controller.signal as any).reason,
```

Comment:
can this be user specified? is it safe to log directly? should it be tagged?

Reply:
we should look into that for all the logged cases.

Reply:
`can this be user or developer specified? is it safe to log directly? should it be tagged?`

For `DeltaManager`, all the reasons are hardcoded. For `runWithRetry`, it could potentially be coming from developer code since it is an exported method.

I can only see taggin as `CodeArtifact`, because if we tag as `UserData` I think it defeats the purpose of this task.